components/xsd-fu/python/ome/modeltools/property.py

```diff
@@ -415,6 +432,17 @@ def _get_isReference(self):
         _get_isReference,
         doc="""Whether or not the property is a reference.""")
 
+    def _get_concreteClasses(self):
+        returnList = []
+        if self.model.opts.lang.hasSubstitutionGroup(self.name):
+            for o in sorted(self.model.objects.values(), lambda x, y: cmp(x.name, y.name)):
+                if o.parentName == self.name:
+                    returnList.append(o.name)
+        return returnList
+    concreteClasses = property(
+        _get_concreteClasses,
+        doc="""Concrete instance types for an abstract type.""")
+
```

Comment:
The get_concreteClasses function must have snuck in with one of my commits to revert to an older schema, it isnt being used and can be removed